test-support/src/main/java/org/springframework/ldap/test/TestContextSourceFactoryBean.java

```diff
@@ -23,12 +23,31 @@
 import org.springframework.ldap.core.support.DefaultDirObjectFactory;
 import org.springframework.ldap.core.support.LdapContextSource;
 import org.springframework.ldap.support.LdapUtils;
+import org.springframework.util.CollectionUtils;
+
+import javax.naming.Name;
+import javax.naming.NamingEnumeration;
+import javax.naming.directory.BasicAttribute;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.ModificationItem;
+import javax.naming.directory.SearchControls;
+import javax.naming.directory.SearchResult;
+import java.util.List;
 
 /**
- * @author Mattias Hellborg Arthursson
+ * @author Mattias Hellborg Arthursson Zilic
```

Comment:
Thank you for pointing out my mistake.
I was mistook that file had three authors before my changes.